packages/devextreme/js/ui/toolbar.d.ts

```diff
@@ -237,7 +237,7 @@ export type Options<
 // eslint-disable-next-line import/first
 import { CheckedEvents } from '../core';
 
-type FilterOutHidden<T> = Omit<T, 'onFocusIn' | 'onFocusOut' | 'onItemDeleted' | 'onItemDeleting' | 'onItemReordered' | 'onSelectionChanged'>;
+type FilterOutHidden<T> = Omit<T, 'onFocusIn' | 'onFocusOut' | 'onItemDeleted' | 'onItemDeleting' | 'onItemReordered' | 'onSelectionChanging'>;
```

Comment:
Do u think toolbar is ready for new onSelectionChanged?))

Reply:
https://github.com/DevExpress/DevExtreme/pull/27954/commits/ca78c9791306c8ad783754bf6230ebb18e85db4b